tests/plot_node_load.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Could you attach a sample output of this script to this PR?

Reply:
[Done](https://github.com/microsoft/CCF/pull/1480#issuecomment-672850824)